i6_models/assemblies/conformer/conformer_v2.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import annotations
```

Comment:
At some point, maybe in the (missing) docstring of the module, you should very briefly explain what's new in `conformer_v2` (just bullet points).

Reply:
they are now added to the doctoring of the module